check_pgactivity

```diff
@@ -552,24 +547,33 @@ sub find_hostkey($) {
 #  * the ref of the structure to save
 #  * the path to the file storage
 sub save($$$$) {
```

Comment:
I think it would be good to document a bit more what `save()` is doing to make sure that you don't overwrite concurrent changes.